### PR TITLE
docs: alexa custom tts announcement

### PR DESCRIPTION
**Pull request by space-pope**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

A pointer to the Alexa custom TTS skill, framed as a news/announcement piece and marked as draft until we're ready to release. Could probably use a hero image.

### CODE REVIEW

File: content/blog/alexa-custom-tts/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Now that we've launched the Maker subscription, we have even more to offer in the smart speaker ecosystem. There's no reason users should be locked in to hearing the same voice in every skill. The sound of a skill's voice should be recognizable, and Maker TTS makes that possible, even for an independent skill developer without the budget for voice talent.
+
+To make experimenting with custom TTS on Alexa dead simple, we've created [a sample Alexa skill](https://github.com/spokestack/alexa-custom-tts) that has all the code you'll need to use Spokestack's TTS in a (very) simple skill. It's completely free to use even without a Maker account, though you'll be limited to Spokestack's free voice until you upgrade.
+
+Installation instructions are in the readme, but as a brief summary, this code uses Amazon's [Alexa-hosted skill](https://developer.amazon.com/en-US/docs/alexa/hosted-skills/build-a-skill-end-to-end-using-an-alexa-hosted-skill.html) feature to copy the code from GitHub into a free sandbox inside Amazon Web Services that runs it on-demand. You just have to change a few lines at the top of one file to include your Spokestack API ID and secret key, and you're good to go.
```
Comment (noelweichbrodt) [inline content/blog/alexa-custom-tts/index.md]: That last line should actually be the last line in the previous paragraph imo. Super important to emphasize the low-code nature of this integration.

File: content/blog/alexa-custom-tts/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The skill itself doesn't do much — it offers a quick bit of encouragement and exits immediately — but all the code is right there and follows a straightforward, familiar pattern from Amazon's numerous samples, so it's straightforward to experiment and build a "real" skill little by little. You can also take the code for calling Spokestack's TTS service and drop it right into an existing skill.
```
Comment (noelweichbrodt) [inline content/blog/alexa-custom-tts/index.md]: maybe make a suggestion for a couple easy ways to expand the skill? eg "Try adding a feature that picks randomly from a list of exhortations each launch. And a way to ask for another encouragement while you're in the skill."

**Comment by timmywil** [comment]
@roppem9 could probably help here with a hero image. Currently set to `blog.png`, which doesn't exist.

File: content/blog/alexa-custom-tts/index.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: Why should your Alexa skill sound like all the others? Use Spokesta
 author: josh
 tags: News, 3rd Party Integration
 draft: true
-hero: blog.png
+hero: ../../assets/alexa-custom-tts.png
```
Comment (space-pope) [inline content/blog/alexa-custom-tts/index.md:8]: This was the logo for the minecraft app...I see the Alexa speech bubble thing, but I'm not sure minecraft fits here. Do we use the plain Alexa logo elsewhere? If so, that seems a bit more natural here.

Comment (roppem9) [reply content/blog/alexa-custom-tts/index.md:8]: Just switched out the logos

**Pull request merged**